calligraphy/src/main/java/io/github/inflationx/calligraphy3/Calligraphy.java

```diff
@@ -103,6 +107,16 @@ public Calligraphy(CalligraphyConfig calligraphyConfig) {
         this.mAttributeId = new int[]{calligraphyConfig.getAttrId()};
     }
 
+    @NonNull
+    private static String getResourceEntryName(@NonNull View view) {
+        try {
+            return view.getResources().getResourceEntryName(view.getId());
+        } catch (Resources.NotFoundException exception) {
+            Log.w(TAG, "An error occurred when trying to get resource entry name. \nStackTrace:" + exception);
+            return "";
+        }
+    }
+
     /**
      * Handle the created view
```

Comment:
``` Suggestion: comment in only one line ```

Reply:
@lucalves I am not sure why you suggesting this change here. Javadoc has not changed and it's not part of this pull request. Won't change.